### ISSUE TITLE
Types path in package.json is missing "./"

### ISSUE THREAD

**Issue by Virenbar** [typpo/quickchart-js#31]
Types path is missing "./" at the beginning.
Because of this the declaration file can't be found.
https://github.com/typpo/quickchart-js/blob/e19e661dcadb51a99caed9db847767414fb8bb37/package.json#L9-L15
![Code_22 06 16-15 34 49](https://user-images.githubusercontent.com/10138491/174052639-04a31486-0188-4545-b566-983ce925e896.png)

**Comment by Virenbar** [comment]
But even after adding "./" and importing:
```ts
import QuickChart from "quickchart-js";
const QC: QuickChart = new QuickChart();
```
I get errors:  
`Cannot use namespace 'QuickChart' as a type.ts(2709)` and `This expression is not constructable. Type 'typeof import("D:/VCS/ARKDiscordBot/node_modules/quickchart-js/build/typescript/index")' has no construct signatures.ts(2351)`
 I can import only like this:
```ts
import { default as QuickChart } from "quickchart-js";
const QC: QuickChart = new QuickChart();
```

**Comment by typpo** [comment]
Which version of the library are you using?  I can't repro locally, nor can I repro in a [third party environment](https://replit.com/@trynafunction/QuickChart-Typescript-Example?v=1)

**Comment by Virenbar** [comment]
I have tried using version 3.0.1.
[Here is branch](https://github.com/Virenbar/ARKDiscordBot/tree/qc-test) with just quickchart
And warning from VS Code in package.json
![image](https://user-images.githubusercontent.com/10138491/174230908-fe00c04c-db00-4291-96d7-f7fedc4f7938.png)
Currently using 2.0.3 which works fine.

**Comment by typpo** [comment]
Thanks for providing the example.  It looks like the immediate fix for v3.0.1 is to set `"moduleResolution": "node"` in your `tsconfig.json` instead of `node16`.
